Unix_HW/tail.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument('file', nargs='*')
+parser.add_argument('-n', '--numbers', action='store')
+
+args = parser.parse_args()
+
+
+if args.numbers:
+    N = int(args.numbers)
```

Comment:
Не надо большую букву
```suggestion
    n_lines = int(args.numbers)
```